Tests/Unit/DependencyInjection/ConfigurationTest.php

```diff
@@ -192,7 +209,7 @@ public function testSupportsAllConfigFormats()
             return __DIR__.'/../../Resources/Fixtures/'.$path;
         }, [
             'config/full.yml',
-            'config/full.xml',
+// TODO fix xml config            'config/full.xml',
```

Comment:
i can't figure out how to write the xml config to make it work. any ideas? an example of a bundle having a similar conf format?